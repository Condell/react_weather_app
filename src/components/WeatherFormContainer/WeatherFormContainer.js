import React, { Component } from "react";

import WeatherForm from './components/WeatherForm/WeatherForm'

class WeatherFormContainer extends Component {
  state = { text: "" };

  handleChange = e => {
    this.setState({ text: e.target.value });
  };

  render() {
    return (
      <WeatherForm onChange={this.handleChange} {...this.props}/>
    );
  }
}



export default WeatherFormContainer;
