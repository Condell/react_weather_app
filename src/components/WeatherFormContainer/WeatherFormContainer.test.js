import React from "react";
import { shallow } from "enzyme";
import 'jest-styled-components'
import toJson from "enzyme-to-json"

import WeatherFormContainer from "./WeatherFormContainer";


describe("WeatherFormContainer", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<WeatherFormContainer />);
  });

  it("should render", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});