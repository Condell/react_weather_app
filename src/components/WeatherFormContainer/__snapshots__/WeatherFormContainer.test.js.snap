// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherFormContainer should render 1`] = `
<WeatherForm
  onChange={[Function]}
  text=""
/>
`;
